queries.sql

```diff
@@ -38,4 +38,88 @@ WHERE name NOT LIKE 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * 
 FROM animals
-WHERE weight_kg >= 10.4 AND weight_kg <= 17.3; 
\ No newline at end of file
+WHERE weight_kg >= 10.4 AND weight_kg <= 17.3; 
+
+-- Inside a transaction update the animals table by setting the 
+-- species column to unspecified. Verify that change was made. 
+-- Then roll back the change and verify that species columns
+--  went back to the state before transaction.
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Kindly verify that the change was made before and after the rollback is made as stated in the requirements. e.g
```sql
BEGIN; -- start transaction

UPDATE animals
SET species = 'unspecified'; -- make change

SELECT species from animals; -- verify that change was made
ROLLBACK;

SELECT species from animals; -- verify that change was undone
```